mysql/mysql/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Hmm.

It doesn't seem very appealing for each image to have its own LICENSE. I don't see one in the repo for the rhel on OpenShift image; was this an accident?